pkgs/by-name/co/cosmic-comp/package.nix

```diff
@@ -77,6 +77,14 @@ rustPlatform.buildRustPackage rec {
     "-Wl,--pop-state"
   ];
 
+  makeFlags = [
+    "prefix=$(out)"
+    "CARGO_TARGET_DIR=target/${stdenv.hostPlatform.rust.cargoShortTarget}"
+  ];
+
+  # Use default stdenv installPhase, not the buildRustPackage one.
+  installPhase = "installPhase";
```

Comment:
Just curious, was there any reason to do this over `dontCargoInstall = true`? (asking since I'd done the latter, but I don't think that should be different from what you ended up doing)

Reply:
When I learned this trick, that option didn't exist :)